src/routes/routes.tsx

```diff
@@ -11,6 +12,10 @@ const router = createBrowserRouter([
     path: "/login",
```

Comment:
não teremos mais a rota /login, a página inicial da nossa aplicação será a página de login.
```suggestion
    path: "/",
```